taskcluster/utils.py

```diff
@@ -1,11 +1,11 @@
 import re
 import json
 import datetime
-import uuid
 import base64
 import logging
 import os
 import requests
+import slugid as slugid_
```

Comment:
I don't think you need to use "as slugid_" here. It doesn't collide with other variables. Just wanted it have it more distinguishable from slugId()?